### PR TITLE
(PA-2183) Update acceptance tests to work with Beaker 4

### PR DESCRIPTION
**Pull request by ekinanp**

### CODE REVIEW

**Comment by ekinanp** [comment]
This is blocked on puppetlabs/beaker#1551, puppetlabs/beaker-puppet#77 and a beaker + beaker-puppet release.

**Comment by ekinanp** [comment]
The maint commit's to get the commit checker in CI to pass. I can go ahead and file a corresponding PUP ticket if that is preferred, just did not see the worth in doing so for a small change (and especially when it is related to PA-2183).

**Comment by ekinanp** [comment]
https://jenkins-master-prod-1.delivery.puppetlabs.net/view/__experimental/job/experimental_puppet-agent-extra_puppet-agent-integration-suite_adhoc-1.10.x/2/ shows a passing ad-hoc run, so this is good to go.

File: acceptance/lib/helper.rb [hunk 1]
```diff
@@ -1 +1,3 @@
 $LOAD_PATH << File.expand_path(File.dirname(__FILE__))
+
```
Comment (speedofdark) [inline acceptance/lib/helper.rb]: I understand what this is doing (I think :)), but there's no specific commentary as to why these changes are required. Does it make sense to add some (brief) info?

Comment (ekinanp) [reply acceptance/lib/helper.rb]: It's more of a compatibility change than a something someone will maintain change, so I think the title covers it.

**Comment by speedofdark** [comment]
Damn reboot.

**Comment by ekinanp** [comment]
I removed the maint commit title as I think it's good to the ticket number and this is not worth creating an extra PUP ticket for. Travis' commit checker can fail for this one.

**Comment by joshcooper** [comment]
Next time can you not merge this when the commit checker is failing, otherwise, it will keep failing each time you merge up. It's fine to put `(maint)` on there and reference the PA ticket in the summary or description.
